optimizely/event/processor.go

```diff
@@ -56,21 +56,71 @@ const DefaultEventFlushInterval = 30 * time.Second
 
 var pLogger = logging.GetLogger("EventProcessor")
 
+// QPOption is the QueuingProcessor options that give you the ability to add one more more options before the processor is initialized.
+type QPConfigOption func(qp *QueueingEventProcessor)
+
+func ProcessorBatchSize(bsize int) QPConfigOption {
```

Comment:
nit: Not sure we need to prepend `Processor` to each of these. Seems redundant since the file name is already processor

Reply:
I thought there might be a class with dispatcher.  but, you may be right.